src/style.scss

```diff
@@ -39,79 +42,83 @@
   height: 100%;
   position: fixed;
   z-index: 1;
-  top: 2vh;
-  left: .5vh;
+  top: 2rem;
+  left: 0.5rem;
   background-image: url(https://bitforbit.notquite.se/files/bitforbit-Logo.eps);
-  background-size:100% auto;
+  background-size: 100% auto;
   background-repeat: no-repeat;
 }
 
-.logo img{
+.logo img {
   width: 100%;
   height: auto;
   margin: 30px 0 0 0;
 }
 
-.form{
+.form {
   position: fixed;
   z-index: 1;
   top: 0;
   right: 0;
   padding: 5px;
   font-family: 'GTPressura-Bold';
-  font-size: 2.5vh;
```

Comment:
btw, `vh` is hundredths of the viewport height, and isn't really an appropriate unit for most things. I changed it to `rem` which goes off the font size instead, would you mind resizing your stuff using that?

Reply:
yes I will do this!